### DISCUSSION TITLE
grouping collections and data models

### DISCUSSION THREAD

**Discussion by thefrontendguy** [directus/directus#11010] [Archive]
It would be nice, if it would be possible to group the collections and the data models in the admin ui.

**Comment by benhaynes** [comment]
How so? You can currently add folders/nesting in the data model settings.

**Reply by thefrontendguy** [reply]
I updated to the latest version. You are right. Now I found it and it is exactly what I was looking for.
Did not see it the first time I was looking.

**Reply by benhaynes** [reply]
Haha,  I came here to respond to your other message, which you must have edited or deleted. Glad you found the folders! We're working actively on improving our documentation, which will help understand these features without opening tickets. ❤️
